apps/api/tests/assistant_update_proposals_m133_rls_test.go

```
@@ -0,0 +1,1376 @@
+		}
+		return nil
+	}); err != nil {
+		t.Fatalf("site-must-belong-to-tenant: %v", err)
+	}
+
+	// THE SECOND-ORDER CONSEQUENCE. With the pair unbound, one organisation's
+	// proposal could be reached by a cascade from another organisation's site
+	// delete. Bound, deleting tenant B's site cannot touch tenant A's rows,
+	// because tenant A's rows can never have named it.
+	deleteAdmin := connectAdmin(t, pool)
+	if _, err := deleteAdmin.Pool.Exec(ctx, `DELETE FROM sites WHERE id = $1`, siteB); err != nil {
```

golangci-lint (2.13.2)

[error] QF1008: could remove embedded field "Pool" from selector

(staticcheck)